src/parser/deathknight/blood/modules/talents/Heartbreaker.js

```diff
@@ -8,13 +8,10 @@ import RESOURCE_TYPES from 'game/RESOURCE_TYPES';
 class Heartbreaker extends Analyzer {
   rpGains = [];
   hsCasts = 0;
-  deathStrikeCost = 45;
+  deathStrikeCost = 40;
```

Comment:
Might be a good idea to make this a const for this module, with deathStrikeCost always being 40RP